test-data/unit/check-flags.test

```diff
@@ -1137,3 +1137,34 @@ strict_equality = True
 [[mypy-b]
 strict_equality = False
 [builtins fixtures/bool.pyi]
+
+[case testNoImplicitReexport]
+# flags: --no-implicit-reexport
+from other_module_2 import a
+
```

Comment:
I'd remove all the blank lines from the test cases except the blank line *between* test cases (so no blank lines between files within a test case).